tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -95,7 +101,7 @@ public void Should_warn_of_bad_assembly_files()
                 }
 
                 writer.Flush();
-                actualMessage = writer.Encoding.GetString(stream.GetBuffer());
+                actualMessage = writer.Encoding.GetString(stream.ToArray());

```

Comment:
More things I don't know! Cool.

> Note that the buffer contains allocated bytes which might be unused. 
> …
> To obtain only the data in the buffer, use the `ToArray` method; however, `ToArray` creates a copy of the data in memory.

Thanks for educating me, @thomaslevesque.